app/assets/scripts/views/ThreeW/filter.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import Faram from '@togglecorp/faram';
+
+import Select from '../../components/form-elements/new-components/select';
```

Comment:
Can we avoid things like `new-components` ? Is there a better way to separate things out? `new-components` will no longer make sense down the line. Ideally, would we replace all the current Selects with the new Select and then we no longer need this? Am just a little bit worried about this getting confusing - it's ok for now if we are transitioning and don't want to make too many changes in a single PR, but in this case, can we add a quick README or so to the `new-components` folder to just explain why it exists, and make a note that ideally sometime down the line we'd consolidate the Selects on the site to this and then no longer need the `new-components` folder?

Reply:
This is temporary and I'll think of something better